data/advanced_preferences.cfg

```diff
@@ -28,6 +28,13 @@
     default=yes
 [/advanced_preference]
 
+[advanced_preference]
+    field=save_file_includes_replay
+    name= _ "Include replay information in save files"
```

Comment:
Maybe add a `description=_""` that explains what this setting does in more words?